e2e-test/src/test/features/Cloudsqlmysql_sink_runtime.feature

```diff
@@ -0,0 +1,120 @@
+Feature: BigQuery to CloudSqlMySql and GCS to CloudSqlMySql Runtime
+
+  @CLDMYSQL @TC-Runtime-BigQuery-to-CLOUDSqlMYSQL:
+  Scenario: TC-CLDMYSQL-RNTM-01:Verify user is able to transferred data from BigQuery to CloudSqlMysql with mandatory fields
+    Given Open DataFusion Project to configure pipeline
+    When Source is BigQuery
+    When Target is CloudSQLMySQL
+    Then Link BigQuery to CloudSQLMySQL to establish connection
+    Then Enter the Source BigQuery Properties for table "clsTableNameBQ"
+    Then Validate the Schema
+    Then Verify the Connector status
+    Then Close the BigQuery Properties
+    Then Enter the sink CloudSQLMySQL Properties for table "clsTableNameBQCS"
+    Then Click on Validate button
+    Then Verify the Connector status
+    Then Close the CloudSQLMySQL Properties
+    Then Save the pipeline
+    Then Preview and run the pipeline
+    Then Verify the preview of pipeline is "success"
+    Then Click on PreviewData for CloudSQL MySQL
+    Then Verify Preview output schema matches the outputSchema captured in properties
+    Then Close the Preview
+    Then Pre records count from CloudSQLMySQL table "clsTableNameBQCS"
+    Then Deploy the pipeline
+    Then Run the Pipeline in Runtime
+    Then Wait till pipeline is in running state
+    Then Open and capture Logs
+    Then Verify the pipeline status is "Succeeded"
+    Then Post records count from CloudSQLMySQL table "clsTableNameBQCS"
+    Then Validate the output record count
+    Then Validate successMessage is displayed
+    Then Validate the count of records transferred from BigQuery "clsTableNameBQ" to CloudSqlMySql "clsTableNameBQCS"
```

Comment:
In addition to checking the number of records transferred, also validate that the correct data is written to the sink in all the test cases